src/App.js

```diff
@@ -32,10 +40,27 @@ const App = ({ initialPath }) => {
       } = window;
       const params = new URLSearchParams(nextSearch);
       setSearch(nextSearch);
-      setThemeName(params.get('theme'));
+      const paramsTheme = params.get('theme');
+      // Note: URL and retrieve theme behavior still TBD
+      if (paramsTheme) {
+        setThemeName(paramsTheme);
+      } else {
+        setThemeName(themeName);
+      }
     }
   }, []);
 
+  React.useEffect(() => {
+    if (typeof window !== 'undefined') {
+      window.localStorage.setItem('theme-name', JSON.stringify(themeName));
```

Comment:
On first render, the cookie isn't getting assigned so I'm seeing something like this (missing descriptive text and missing icon from hpe button)

<img width="1009" alt="Screen Shot 2022-07-15 at 2 51 22 PM" src="https://user-images.githubusercontent.com/12522275/179315930-e771b75f-fd37-4d05-b5fb-905b31392745.png">